Makefile

```diff
@@ -174,40 +174,40 @@ $(BUILD_DIRECTORY):
 $(BUILD_TEMPORARY_DIRECTORY): | $(BUILD_DIRECTORY)
 	mkdir $@
 
-$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies: | $(BUILD_DIRECTORY)
-	mkdir $@
-
-$(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies: | $(BUILD_DIRECTORY)
-	mkdir $@
-
 $(BUILD_OUTPUT_DIRECTORY): | $(BUILD_DIRECTORY)
 	mkdir $@
 
-$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules: package.json npm-shrinkwrap.json \
-	| $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies
+$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies: package.json npm-shrinkwrap.json \
+	| $(BUILD_DIRECTORY)
+	mkdir $@
+	cp -rf src $@
 	./scripts/build/dependencies-npm.sh -p \
 		-r "$(TARGET_ARCH)" \
 		-v "$(ELECTRON_VERSION)" \
-		-x $| \
+		-x $@ \
 		-t electron \
 		-s "$(TARGET_PLATFORM)"
+	rm -rf $@/src
```

Comment:
The issue is that installing the dependencies require the source code to be present in the same directory, however we can't make any assumption in the script that `src` is the correct directory, or that its even the only one.

Regarding naming, `src` and `lib` are pretty standard directory names in the node community. `src` contains native code, and `lib` contains JavaScript code.

Hm, actually, given that `src` is pretty standard I think we *can* make that assumption.